### PR TITLE
Prepare v 1.3

### PR DESCRIPTION
**Pull request by spapas**

### CODE REVIEW

**Comment by spapas** [comment]
Hello @treyhunner I've prepared a new version (1.3). This includes all recent PRs and also has Django 4.1 support (since it was more or less "free", no need to change anything I just added it to the tox). 

I've changed all things that I thought needed changing for the new version release, however I  am not sure if I've missed anything...

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ build
 docs/_build
 MANIFEST
 .vscode
+testfiles
```
Comment (treyhunner) [inline .gitignore:13]: It looks like your editor stripped the final newline. You may want to install an [EditorConfig plugin](https://github.com/treyhunner/django-email-log/blob/main/.editorconfig) for your editor to pick up the settings in [this file](https://github.com/treyhunner/django-email-log/blob/main/.editorconfig) automatically.

**Review by treyhunner — APPROVED** [review]
This looks great to me @spapas!

I made a comment about whitespace styling an EditorConfig. But that's it.

I think we should merge this, make a `1.3.0` tag on GitHub pointing to the latest commit on `main` at that point, and then release it to PyPI (which I think I'll need to do for now).

File: .gitignore (hunk 1 above)
Comment (spapas) [reply .gitignore]: Hey @treyhunner I have the editorconfig plugin but probably I edited that file (.gitignore) with a different editor so it didn't add the newline :) 

Added a new commit should be fine now!

**Comment by spapas** [comment]
Hello @treyhunner this PR has been merged along with the the 1.3.0 tag so it should be ready for releasing to pypi. Also the documentation will need updating https://django-email-log.readthedocs.io/en/latest/  because it stil points to the old version

Thank you!

Kind regards,
Serafeim

**Comment by treyhunner** [comment]
I ran `make test-release` and saw a minor issue identified by `twine check` which I fixed in a6489911bb350f5b98b6e0308dc5fd57bb978b36.

After `test-release` passed I made that commit and then retagged:

```
git tag -d 1.3.0
git push origin :1.3.0
git tag 1.3.0
git push --tags
```

Then I did a release with `make release`

It looks like it worked as expected: https://pypi.org/project/django-email-log/1.3.0/

Thanks for putting this together @spapas!

**Comment by spapas** [comment]
Hey @treyhunner the pypi release is cool but the docs aren't upgraded with the latest changes :( 

i.e https://django-email-log.readthedocs.io/en/latest/usage.html

vs

https://github.com/treyhunner/django-email-log/blob/main/docs/usage.rst

I'm really not sure how readthedocs.io is working and what it needs to be updated 😕

**Comment by treyhunner** [comment]
I just manually trigger a build there and it seemed to work. I'm not sure why it didn't happen automatically...

**Pull request merged**